decidim-admin/app/views/decidim/admin/areas/edit.html.erb

```diff
@@ -1,3 +1,4 @@
+<% add_decidim_page_title(translated_attribute(area.name)) %>
```

Comment:
This has an inconsistency ... in the `area_types` we set the title to "Edit area type", yet here you are defining as the `translated_title`
```suggestion
<% add_decidim_page_title(t(".title")) %>
```